BEIMA.Client/src/shared/NavBar.js

```diff
@@ -0,0 +1,35 @@
+import logo from './BSU-logo.png';
+import { Row, Col} from 'react-bootstrap';
+import './shared.css';
+
+const NavBar = () => {
+    return (
+        <div className="sharedNavBar">
+            <Col>
```

Comment:
I would wrap these in a "Container" element, according to this documentation: https://react-bootstrap.github.io/layout/grid/